grpc/server.go

```diff
@@ -0,0 +1,87 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/agglayer/aggkit/log"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+type ServerConfig struct {
+	// Host is the address to bind the gRPC server
+	Host string `mapstructure:"Host"`
+	// Port is the port to bind the gRPC server
+	Port int `mapstructure:"Port"`
+	// EnableReflection indicates whether gRPC server reflection is enabled
```

Comment:
If we opt to keep this parameter, instead on relying on production vs development parameter, then it is worth explaining that it enables exposure of service methods, etc. WDYT?

Reply:
Fix: https://github.com/agglayer/aggkit/pull/711/commits/8426eb824e2e249d622e420bc3f19ae221fbf9a0